deploy/charts/rook-ceph/templates/configmap.yaml

```diff
@@ -82,9 +88,23 @@ data:
   ROOK_CSIADDONS_IMAGE: {{ .Values.csi.csiAddons.image | quote }}
 {{- end }}
 {{- end }}
+{{- if .Values.csi.nfs }}
+  ROOK_CSI_ENABLE_CEPNHFS: {{ .Values.csi.nfs.enabled | quote }}
+{{- if .Values.csi.nfs.image }}
+  ROOK_CSI_NFS_IMAGE: {{ .Values.csi.nfs.image | quote }}
+{{- end }}
+{{- if .Values.csi.nfs.probe }}
+{{- if .Values.csi.nfs.probe.image }}
```

Comment:
This double check can be eliminated by using `nfsProbe.image` instead of `nfs.probe.image`. 

Further, is this clearer as`nfsDriverProbe.image`? @Madhu-1 thoughts?